src/CMakeLists.txt

```diff
@@ -540,7 +559,7 @@ find_package(Pugixml 1.2 REQUIRED)
 include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 list(APPEND LIBS ${Pugixml_LIBRARIES})
 
-if(NOT SOURCE_PACKAGE AND NOT APPLE)
+if(NOT SOURCE_PACKAGE AND NOT APPLE AND NOT WIN32)
```

Comment:
Basically I have 2 major types of warning during compile:
* Shadow warnings, like 
```warning: declaration of 'data' shadows a previous local [-Wshadow]
       const uchar8 *data = mFile->getData(key_off, 1);```
* printf format warnings, due to the fact that C runtime (MSVCRT) does not have all the format strings, like **z**. Example:
``` warning: unknown conversion type character 'z' in format [-Wformat=]
  ThrowRDE("Out of memory, failed to allocate %zu bytes", size*sizeof(int));```